### PR TITLE
direct request: document requesters and minContractPaymentLinkJuels

### PR DESCRIPTION
**Pull request by jmank88**
@boxhock noticed that `requesters` and `minContractPaymentLinkJuels` were undocumented. This change adds the examples from the v0.10.14 changelog: https://github.com/smartcontractkit/chainlink/blob/develop/docs/CHANGELOG.md#requestersmincontractpaymentlinkjuels

Towards #382

### CODE REVIEW

File: docs/Oracle Jobs/Job Types/direct_request.md [hunk 1]
```diff
@@ -166,4 +168,4 @@ observationSource   = """
 """
 ```
 
-Multi-word response jobs like this, and also large response jobs need to use the [Operator contract](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.7/Operator.sol) as the on-chain contract that coordinates things between the consuming contract and the Chainlink node, as opposed to the legacy [Oracle contract](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.6/Oracle.sol).
\ No newline at end of file
+Multi-word response jobs like this, and also large response jobs need to use the [Operator contract](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.7/Operator.sol) as the on-chain contract that coordinates things between the consuming contract and the Chainlink node, as opposed to the legacy [Oracle contract](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.6/Oracle.sol).
```
Comment (jmank88) [inline docs/Oracle Jobs/Job Types/direct_request.md:171]: I didn't touch this - maybe just CR-LF :shrug:

File: docs/Oracle Jobs/Job Types/direct_request.md [hunk 2]
```diff
@@ -15,8 +15,15 @@ type                = "directrequest"
...
 contractAddress     = "0x613a38AC1659769640aaE063C651F48E0250454C"
-# Optional externalJobID: Automatically generated if unspecified
-externalJobID       = "0EEC7E1D-D0D2-476C-A1A8-72DFB6633F02"
+
+# Optional fields:
+# requesters        = [
+#   "0xaaaa1F8ee20f5565510B84f9353F1E333E753B7a",
+#   "0xbbbb70F0e81C6F3430dfdC9fa02fB22BdD818C4e"
+# ]
+# minContractPaymentLinkJuels = "100000000000000"
```
Comment (dwightjl) [inline docs/Oracle Jobs/Job Types/direct_request.md:24]: I tested this on a 1.3.0 node. When I add `minContractPaymentLinkJuels` without the `#` character, it doesn't show up in the job definition. It behaves the same way as any other invalid field where the field is completely ignored and doesn't show up in the final job spec definition when you view it in the operators UI. Add `invalidField = "10101"`, for example. Are we sure this field is actually functional?

The `requesters` field works fine and shows up in the definition as expected.

**Review by dwightjl — CHANGES_REQUESTED** [review]

File: docs/Oracle Jobs/Job Types/direct_request.md (hunk 1 above)
Comment (dwightjl) [reply docs/Oracle Jobs/Job Types/direct_request.md:178]: The previous file had no newline at the end, so your editor probably automatically added it. My editor, Atom, does the same. We can ignore this.

**Review by boxhock — APPROVED** [review]
LGTM :+1:

File: docs/Oracle Jobs/Job Types/direct_request.md (hunk 2 above)
Comment (vyzaldysanchez) [reply docs/Oracle Jobs/Job Types/direct_request.md:24]: PR addressing this issue: https://github.com/smartcontractkit/chainlink/pull/6487

Comment (jmank88) [reply docs/Oracle Jobs/Job Types/direct_request.md:24]: @dwightjl Fixed on develop, and it was only a UI limitation so we can probably merge these docs now :shrug:

**Review by dwightjl — APPROVED** [review]

**Pull request merged**